src/pages/Alerts/Alerts.js

```diff
@@ -10,24 +10,60 @@ import PageLoader from '../../components/Loader/PageLoader'
 import { MobileOnly } from '../../components/Responsive'
 import { RecommendedSignals } from '../SonarFeed/SonarFeedRecommendations'
 import AlertModal from '../../ducks/Alert/AlertModal'
+import MyAlertsTab from './MyAlertsTab/MyAlertsTab'
+import AlertRestrictionMessage from './AlertRestrictionMessage/AlertRestrictionMessage'
 import { useSignals } from '../../ducks/Signals/common/getSignals'
 import { useUser } from '../../stores/user'
 import { mapSizesToProps } from '../../utils/withSizes'
 import { prepareAlertTitle } from '../../ducks/Signals/link/OpenSignalLink'
+import { useUserSubscriptionStatus } from '../../stores/user/subscriptions'
 import styles from './Alerts.module.scss'
 
 const LoadableAlertsList = Loadable({
   loader: () => import('../SonarFeed/SignalsList'),
   loading: () => <PageLoader />
 })
 
+function getAlertsRestrictions ({ signals, isPro, isProPlus }) {
+  if (isPro) {
+    const maxAmount = 20
+    return {
+      currentAmount: signals.length,
+      maxAmount,
+      shouldHideRestrictionMessage: signals.length !== maxAmount
+    }
+  }
+  if (isProPlus) {
+    const maxAmount = Infinity
+    return {
+      currentAmount: signals.length,
+      maxAmount,
+      shouldHideRestrictionMessage: true
+    }
+  }
+
+  const maxAmount = 3
+  return {
+    currentAmount: signals.length,
+    maxAmount,
+    shouldHideRestrictionMessage: signals.length !== maxAmount
+  }
+}
```

Comment:
I think it's possible to simplify it to something like this:

```suggestion
  const maxAmount = isProPlus ? Infinity : isPro ? 20 : 3
  return {
    maxAmount,
    currentAmount: signals.length,
    shouldHideRestrictionMessage: isProPlus || signals.length !== maxAmount
  }
}
```